File: csharp/SlimDownYourAggregates.Tests/Slimmed/Events/ChapterAddedEvent.cs
using SlimDownYourAggregates.Tests.Slimmed.Core;
using SlimDownYourAggregates.Tests.Slimmed.Entities;

namespace SlimDownYourAggregates.Tests.Slimmed.Events;

public record ChapterAddedEvent(BookId BookId, Chapter Chapter): IDomainEvent;

File: java/src/main/java/io/eventdriven/slimdownaggregates/slimmed/events/ChapterAddedEvent.java
package io.eventdriven.slimdownaggregates.slimmed.events;

import io.eventdriven.slimdownaggregates.slimmed.core.IDomainEvent;
import io.eventdriven.slimdownaggregates.slimmed.entities.Chapter;

import java.util.UUID;

public record ChapterAddedEvent(UUID bookId, Chapter chapter) implements IDomainEvent {
}
